lib/client/index.js

```diff
@@ -660,6 +660,48 @@ client.init = function init(serverSettings, plugins) {
   ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
   socket = io.connect();
 
+  function mergeTreatmentUpdate(isDelta, cachedDataArray, receivedDataArray) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (11)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1542254962/issues/source?bid=2452379&fileBranchId=3015078#l663)